app/scripts/controllers/preferences.js

```diff
@@ -290,6 +291,18 @@ export default class PreferencesController {
   }
   ///: END:ONLY_INCLUDE_IF
 
+  /**
+   * Setter for the `bitcoinSupportEnabled` property.
+   *
+   * @param {boolean} bitcoinSupportEnabled - Whether or not the user wants to
+   * enable the "Add Bitcoin account" button.
```

Comment:
I used a different wording to match the existing one for Ethereum:
- Was initially "Add a new account" and I changed it to "Add a new Ethereum account"
- Similarly, I used "Add a new Bitcoin account" for Bitcoin

Do you think we can update this one too?

Reply:
[324b955](https://github.com/MetaMask/metamask-extension/pull/25672/commits/324b95568b5c1ffe5b505cebfdf4c0e3820a7127)